nes_bnf.html

```diff
@@ -0,0 +1,1204 @@
+          <span class="definition">NazwaJęzyka</span> ::= { nazwa języka; podawać tylko w przypadku, gdy KodJęzyka nie jest jednoznaczny }
+        </div>
+
+        <div class="blockDef" id="Lokalizacja">
+          <span class="definition">Lokalizacja</span> ::= <span class="ptag">lokalizacja</span> { lokalizacja dokumentu - do usunięcia ? }
+                      <span class="ktag">lokalizacja</span>
+        </div>
+
+        <div class="blockDef" id="Status">
+          <span class="definition">Status</span> ::= <span class="ptag">status</span> { status dokumentu - pole do usunięcia ? }
+                      <span class="ktag">status</span>
+        </div>
+
+        <div class="blockDef" id="DataDnia">
+          <span class="definition">DataDnia</span> ::= <span class="ptag">data</span> <span class="useHere">Rok</span>
+                      - <span class="useHere">Miesiąc</span> - <span class="useHere">Dzień</span>
+                      <span class="ktag">data</span>
+        </div>
+
+      </div>
+
+    </div>
+
+    <script>
+
+      function setTagsOnClicks (span_class, js_call ) {
+          var all_span = document.getElementsByTagName("span");
+          for (var i=0, max=all_span.length; i < max; i++) {
+              var elem = all_span[i];
+              if ( elem.className == span_class ) {
+                  elem.setAttribute("onClick",js_call);
+              }
+          }
+      }
+
+      function appendDivToAllDefinitionStruct(main_div_class, new_tag, new_tag_attr, new_tag_attr_value ) {
+          var all = document.getElementsByClassName(main_div_class);
+          for (var i=0, max=all.length; i < max; i++) {
+              var node = document.createElement(new_tag);
+              node.setAttribute(new_tag_attr,new_tag_attr_value);
+              all[i].appendChild(node);
+          }
+      }
+
+      function getProperDefObj ( id ) {
+          var definition = null;
+          var all = document.getElementsByTagName("div");
+          for (var i=0, max=all.length; i < max; i++) {
+              var elem = all[i];
+              if ( elem.className == "blockDef" ) {
+                  if ( elem.getAttribute("id") == id ) {
+                      definition = elem;
+                      break;
+                  }
+              }
+          }
+          return definition;
+      }
+
+      function getUsingDefObjs ( name ) {
+          var definitions = [ ];
+          var all = document.getElementsByTagName("div");
+          for (var i=0, all_max=all.length; i < all_max; i++) {
+              var elem = all[i];
+              if ( elem.className == 'blockDef' ) {
+                  var uses_in_elem = elem.getElementsByTagName("span");
+                  for (var j=0, uses_max=uses_in_elem.length; j < uses_max; j++) {
+                      var child = uses_in_elem[j];
+                      if ( child.className == "useHere"  &&
+                           child.innerText == name         ) {
```

Comment:
Specjalnie. Chodziło mi o to, by pionowo optycznie objąć dwa warunki i łączący je operator.
Jeśli nie uważasz tego zapisu za „potworny”, chętnie bym tak zostawił